event_webhook_types.go

```diff
@@ -763,6 +763,7 @@ type PipelineEvent struct {
 		When           string     `json:"when"`
 		Manual         bool       `json:"manual"`
 		AllowFailure   bool       `json:"allow_failure"`
+		FailureReason  string     `json:"failure_reason"`
```

Comment:
Following the ordering from the docs, so can you please insert the new field in between `QueuedDuration` and `When`?